### PR TITLE
docs: Link to finding-browser-versions MDN article

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

**Comment by sideshowbarker** [comment]
> Thanks for this PR! 👍
> 
> The data-guidelines documentation is meant to list preferences for more consistent data and I think how-to material or links on how to contribute better should go to contributing.md

OK, yep — will move this over there

**Comment by sideshowbarker** [comment]
OK, just now pushed another change that moves it to contributing.md

File: docs/contributing.md [hunk 1]
```diff
@@ -32,6 +33,10 @@ There are many ways you can help improve this repository! For example:
 - **Review a pull request:** there is a list of [PRs](https://github.com/mdn/browser-compat-data/pulls).
   Let us know if these look good to you.
 
+## Finding browser version numbers for features
+
+When adding data for a particular feature, you'll often need to find which version of each browser the feature first shipped in. For how-to guidance which will help you do that, see [Matching web features to browser release version numbers](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Processes/Matching_features_to_browser_version).
```
Comment (vinyldarkscratch) [inline docs/contributing.md:38]: Perhaps de-localize the URL?

```suggestion
When adding data for a particular feature, you'll often need to find which version of each browser the feature first shipped in. For how-to guidance which will help you do that, see [Matching web features to browser release version numbers](https://developer.mozilla.org/docs/MDN/Contribute/Processes/Matching_features_to_browser_version).
```

Comment (sideshowbarker) [reply docs/contributing.md]: > Perhaps de-localize the URL?

Yup — thanks!

**Pull request merged**